modules/monitoring/models/host.php

```diff
@@ -285,6 +285,9 @@ public function check_now() {
 	 * @ninja orm_command view monitoring/naemon_command
 	 */
 	public function acknowledge_problem($comment, $sticky=true, $notify=true, $persistent=true) {
+		if(empty(trim($comment))) {
+			$comment = "Acknowledged by <".$this->get_current_user().">";
```

Comment:
Maybe it would be nicer without the `<` and `>` around the username? I think they were just meant as placeholders in the jira issue.